src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -59,22 +71,41 @@ public PersonCard(Person person, int displayedIndex) {
         this.person = person;
         id.setText(displayedIndex + ". ");
         name.setText(person.getName().fullName);
-        phone.setText(person.getPhone().value);
-        address.setText(person.getAddress().value);
-        email.setText(person.getEmail().value);
-        subjectWithGrade.getChildren().add(new Label(person.getSubject().value));
-        subjectWithGrade.getChildren().add(new Label(person.getGrade().value));
-        attendance.setText(person.getAttendance().value);
-        payment.setText(person.getPayment().value);
-        dateTimes.setHgap(5);
-        person.getDateTimes().stream()
-                .sorted(Comparator.comparing(dateTime -> dateTime.value))
-                .forEach(dateTime -> dateTimes.getChildren()
-                        .add(new Label(LocalDateTime.parse(dateTime.value,
-                                        DateTimeFormatter.ofPattern("uuuu-MM-dd HHmm"))
-                                .format(DateTimeFormatter.ofPattern("MMM d uuuu h:mma")))));
         person.getTags().stream()
                 .sorted(Comparator.comparing(tag -> tag.tagName))
                 .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+        setField(phone, PHONE_DESCRIPTION, person.getPhone().value);
```

Comment:
Use of descriptive labels in the UI is a good idea. It enhances clarity and makes the interface more user-friendly. Great job!